src/MyEventEmitter.js

```diff
@@ -1,14 +1,70 @@
 'use strict';
 
 class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  constructor() {
+    this.listeners = {};
+  }
+
+  on(event, callback) {
+    if (!this.listeners[event]) {
+      this.listeners[event] = [];
+    }
+
+    this.listeners[event].push(callback);
+  }
+
+  once(event, callback) {
+    const wrapper = (...args) => {
+      callback(args);
```

Comment:
The `callback` function is called with an array `args` instead of spreading the arguments. Consider using `callback(...args)` to pass individual arguments to the callback.